packages/flame/src/InputGroup/InputGroup.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { themeGet } from '@styled-system/theme-get';
+import { color, ColorProps, zIndex, ZIndexProps, compose } from 'styled-system';
+
+import { Flex, border, FlameFlexProps, BorderProps } from '../Core';
+
+export interface InputGroupAddonProps
+  extends FlameFlexProps,
+    BorderProps,
+    Partial<Omit<ColorProps, 'color'>>,
+    ZIndexProps {}
+const InputGroupAddon = styled(Flex)<InputGroupAddonProps>`
+  padding: ${themeGet('space.1')} ${themeGet('space.2')};
+  text-align: center;
+  background-color: ${themeGet('groupStyles.addon.background')};
+  border: solid 1px ${themeGet('groupStyles.addon.borderColor')};
+  border-radius: ${themeGet('radii.radius-1')};
+  align-items: center;
+  ${compose(
+    border,
+    color,
+    zIndex,
+  )};
+`;
+
+const InputGroup: React.FC = ({ children, ...restProps }) => {
+  const nextChildren = React.Children.map(children, (child: any, index) => {
```

Comment:
`any` => `React.Element`?

Reply:
Usually there is a good reason @xdrdak uses `any` for those. I'd keep it as is and fix in a later PR.

Reply:
Maybe sometimes it's only a text, in such case I'm not sure it's a `React.Element`. But we should remove any `any` in the future.